geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DefineIndexCommand.java

```diff
@@ -49,15 +50,15 @@ public Result defineIndex(
           help = CliStrings.DEFINE_INDEX__TYPE__HELP) final String indexType) {
 
     Result result;
-    int idxType;
+    IndexType idxType;
 
     // Index type check
     if ("range".equalsIgnoreCase(indexType)) {
-      idxType = IndexInfo.RANGE_INDEX;
+      idxType = IndexType.FUNCTIONAL;
```

Comment:
Same comment about using a Stream applies here.

Reply:
this will get deleted with the next PR by changing the parameter type from String to this enum IndexType for better client side validation.